### ISSUE TITLE
How to add disk?

### ISSUE THREAD

**Issue by wxf26054** [kroese/virtual-dsm#311]

**Comment by Roalkege** [comment]
@kroese, but I can't create a shared folder with an SMB/NFS drive.
Is there an option for this? So to make a network drive a shared folder? Would this be possible with the shell and links?

**Comment by Roalkege** [comment]
Yeah I can add a CIFS but I cant use the folder for Hyper Backup.

I can only Backup to a shared Folder.

**Comment by binsir1** [comment]
> Yeah I can add a CIFS but I cant use the folder for Hyper Backup.
> 
> I can only Backup to a shared Folder.

This individual intends to pass Unraid's shared folders directly into DSM. In the past, Docker containers only required adding the Unraid folder path to be usable within the container. This facilitates establishing a connection between Unraid and DSM. How can this be achieved?

**Comment by binsir1** [comment]
> > Yeah I can add a CIFS but I cant use the folder for Hyper Backup.
> > I can only Backup to a shared Folder.
> 
> This individual intends to pass Unraid's shared folders directly into DSM. In the past, Docker containers only required adding the Unraid folder path to be usable within the container. This facilitates establishing a connection between Unraid and DSM. How can this be achieved?

This is also the goal I want to achieve.

**Comment by binsir1** [comment]
> 

thanks,bro
